jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/stream/LogFileToAsn1CodecPublisher.java

```diff
@@ -82,7 +83,19 @@ private void publish(XmlUtils xmlUtils) throws JsonProcessingException {
          OdeDriverAlertPayload driverAlertPayload = new OdeDriverAlertPayload(((DriverAlertFileParser) fileParser).getAlert());
          OdeDriverAlertMetadata driverAlertMetadata= new OdeDriverAlertMetadata(driverAlertPayload);
          driverAlertMetadata.getSerialId().setBundleId(bundleId.get()).addRecordId(1);
-         OdeLogMetadataCreatorHelper.updateLogMetadata(driverAlertMetadata, fileParser);
+
+
+         TimLogLocation driverAlertLocation = ((DriverAlertFileParser) fileParser).getLocation();
+         OdeLogMsgMetadataLocation driverAlertMetadataLocation =   new OdeLogMsgMetadataLocation(
```

Comment:
Shouldn't we be converting the location data to their user-friendly lat/long/speed/heading formats? If so, use `Position3DBuilder.odePosition3D(new DsrcPosition3D(latitude, longitude, elevation)` and `SpeedOrVelocityBuilder.genericSpeedOrVelocity()` (you'll need to make it a public method) and `HeadingBuilder` (fine or coarse heading depending on what's in the log file). This comment also applies to TIM messages. They too should possibly be made user-friendly.